### PR TITLE
Set hostname for individual lines. 

### PR DESCRIPTION
**Pull request by vilyapilya**
Read hostname from the sent message and pass with option to LogDNA

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ class BunyanStream extends EventEmitter {
             , app: record.name
             , context: Object.assign({}, record)
             , index_meta: true
+            , hostname: record.hostname
```
Comment (smusali) [inline index.js:41]: as shown in [`logdna/logdna-winston`](https://github.com/logdna/logdna-winston/blob/master/index.js), `hostname` can be given thru `options` - so, I recommend to add `this.hostname = options.hostname` to the `constructor` and change this line to `, hostname: this.hostname || record.hostname`

**Review by smusali — CHANGES_REQUESTED** [review]
small change request

File: index.js (hunk 1 above)
Comment (vilyapilya) [reply index.js:41]: hm... in that case this change is redundant.  You already can set a hostname on the logger level with its current state. The idea is, how I understand and how it work in nodejs, is to be able to set an optional hostname on the line level

Comment (smusali) [reply index.js:41]: @vilyapilya, I agree with that but what if a customer wants to have all lines having the same `hostname` - `record.hostname` does always exist but a customer may give optional `hostname` thru the constructor to override all `hostname`s; I mean, if `options` contains `hostname`, that means a customer wants to see that as a `hostname` for all lines - if it doesn't contain, then by default, it's always going to be `record.hostname`

**Review by smusali — CHANGES_REQUESTED** [review]
just wondering whether it's possible to do this in `this.hostname || record.hostname` way

File: index.js (hunk 1 above)
Comment (vilyapilya) [reply index.js:41]: in that case, he adds the hostname to the options object he passes

Comment (smusali) [reply index.js:41]: Yup - now I see - thanks for clarification!

**Review by smusali — APPROVED** [review]
lgtm!

**Pull request merged**